src/core/Item.java

```diff
@@ -0,0 +1,116 @@
+package core;
+
+import commands.Command;
+
+import java.io.File;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Marcelo Loayza on 17/05/2016.
+ */
+public class Item {
+
+    private String name;
+    private String location;
+    private String type;
+    private String fileSize;
+    private Date lastModified;
+    private String filePermissions;
+    private Map<String, String> map;
+
+    public Item (File fileSystemItem) {
+        map = new HashMap<String, String>();
```

Comment:
map??? this is too generic? what values holds, what is the purpose of this variable in the context of the method, please rename it and put a more appropiate name for this variable